### ISSUE TITLE
build-tarballs makes a tarball with a cached envirionment pickle in it

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7980]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7980|
|Type|defect|
|Created|2015-07-24 11:40:16Z|
|Branch|https://github.com/twisted/twisted/tree/nopickledenv-7980|

```twisted.python._release.SphinxBuilder``` needs to give the -d option to save the environment somewhere else, or maybe -E to not create it at all. This should cut down on the tarball size a bit.

<details><summary>Searchable metadata</summary>

```
trac-id__7980 7980
type__defect defect
reporter__hawkowl hawkowl
priority__highest highest
milestone__None None
branch__branches_nopickledenv_7980 branches/nopickledenv-7980
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1437738016073901 1437738016073901
changetime__1445217443410268 1445217443410268
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

From my unscientific tests, it brings it down by about a meg and a half.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

(In [45447]) Branching to nopickledenv-7980.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Item 1 is with this patch, item 2 is the prerelease which didn't have it.

```
-rw-r--r--   1 red   wheel  3111287 10 Aug 21:49 Twisted-15.3.0.tar.bz2
-rw-r--r--   1 red   wheel  4432359 23 Jul 15:09 Twisted-15.3.0pre1.tar.bz2
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Changes look good and make sense. Thanks!

Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45468]) Merge nopickledenv-7980: Don't include the pickled Sphinx environment in release tarballs

Author: hawkowl
Reviewer: adiroiban
Fixes: [#7980](https://github.com/twisted/twisted/issues/7980)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_9"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
